frontend/src/scenes/authentication/signup/verify-email/VerifyEmail.tsx

```diff
@@ -17,28 +19,75 @@ export const VerifyEmailHelpLinks = (): JSX.Element => {
     const { requestVerificationLink } = useActions(verifyEmailLogic)
     const { uuid } = useValues(verifyEmailLogic)
     const { openSupportForm } = useActions(supportLogic)
+    const [iNeedHelp, setINeedHelp] = useState(false)
+    const [checkListValues, setCheckListValues] = useState<boolean[]>([])
+
+    if (!iNeedHelp) {
+        return (
+            <LemonButton type="secondary" className="mt-2" onClick={() => setINeedHelp(true)}>
+                I need help
+            </LemonButton>
+        )
+    }
+
+    const checklist = ['Checked your spam folder', 'Checked with your IT department (if applicable)']
+
+    const handleChecklistChange = (index: number): void => {
+        const newCheckListValues = [...checkListValues]
+        newCheckListValues[index] = !newCheckListValues[index]
+        setCheckListValues(newCheckListValues)
+    }
+
+    const allChecked = checklist.every((_, index) => checkListValues[index])
 
     return (
-        <div className="flex flex-row gap-x-4">
-            <LemonButton
-                type="secondary"
-                className="mt-8"
+        <div className="w-full bg-bg-3000 p-4 rounded relative">
+            <IconX
+                className="absolute top-3 right-3 cursor-pointer"
                 onClick={() => {
-                    openSupportForm({ kind: 'bug', target_area: 'login' })
+                    setCheckListValues([])
+                    setINeedHelp(false)
                 }}
-            >
-                Contact support
-            </LemonButton>
-            {uuid && (
-                <LemonButton
-                    type="secondary"
-                    className="mt-8"
-                    onClick={() => {
-                        requestVerificationLink(uuid)
-                    }}
-                >
-                    Request a new link
-                </LemonButton>
+            />
+            <div className="flex flex-col justify-center">
+                <p className="text-left mb-2">Please confirm you've done the following:</p>
```

Comment:
This need a lot more context.

```suggestion
                <p className="text-left mb-2">Sorry you're having troubles! We're here to help, but first we ask that you check a few things first on your end. Generally any issues with email happen after they leave our hands.</p>
                <p className="text-left mb-2">Please try the following steps to resolve before you reach out:</p>
```

This will fail formatting for line-length, prob will need to pull down and save the file locally to reformat it.